### DISCUSSION TITLE
Generic "images" Table

### DISCUSSION THREAD

**Discussion by danielyogel** [prisma/prisma#8784] [Q&A]
I want to use a generic 'image' table, which is connected by one to one relation to ANY other table.
I don't see how can I do this using Prisma, because Prisma forces me to declare which table table the 'image' table is related to.

How may I solve this? Or, otherwise, any better best practice to use when persisting image meta-data in the DB.

Thanks.

**Comment by janpio** [comment]
You would probably want to put the relation on all the other models, references `Image` via `@relation` (and also adding the relation scalar field on all these tables, e.g. `image_id`). Then that model will have a lot of back relation to all these tables, but that should not be a problem.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
